test/unit/domain/services/state/WarpStateCacheRetentionResults.test.ts

```
@@ -0,0 +1,86 @@
+
+  it('rejects empty report names and errors', () => {
+    expect(() => new WarpStateCacheRetentionReport({
+      liveSnapshotIds: [''],
+      anchoredSnapshotIds: [],
+      unanchoredSnapshotIds: [],
+      missingSnapshotIds: [],
+      wrongTypeSnapshotIds: [],
+      staleRootNames: [],
+      mismatchedRootNames: [],
+      rootSetError: null,
+    })).toThrow(/cannot contain an empty name/);
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)